<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>980</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>miniQQ</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>490</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_all">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="UserChatItem" name="UserChatItemwidget" native="true">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="InputMessageItem" name="InputMessageItemwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>300</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="UserMenuItem" name="UserMenuItemwidget" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="SearchUser" name="SearchUserwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="UserChatListItem" name="UserChatListItemwidget" native="true">
          <property name="minimumSize">
           <size>
            <width>170</width>
            <height>770</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>170</width>
            <height>770</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserMenuItem</class>
   <extends>QWidget</extends>
   <header location="global">UserMenuItem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserChatListItem</class>
   <extends>QWidget</extends>
   <header location="global">UserChatListItem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserChatItem</class>
   <extends>QWidget</extends>
   <header location="global">UserChatItem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InputMessageItem</class>
   <extends>QWidget</extends>
   <header location="global">InputMessageItem.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SearchUser</class>
   <extends>QWidget</extends>
   <header location="global">SearchUser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
